index.html

```diff
@@ -3,13 +3,60 @@
 
 <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">    
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto" rel="stylesheet">
     <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="./style.css">  
 </head>
 
 <body>
+    <header>
+        <div class="header"> 
+            <img class="hamburger" src="img\menu-hamburger.svg" alt="dropdown menu">
+            
+            <p class="sitemoto" >The best cakes in town delivered to your door</p>  
+        </div>
+        <div class="sideimg">
+           <img class="headerimg" src="https://images.immediate.co.uk/production/volatile/sites/30/2020/08/triple-chocolate-peanut-butter-layer-cake-2-06eee24.jpg" alt="cake">  
+          </div>
+    </header>
+    <main>
+        <section class="nav">
+            <ul class="navbar">
+                <li><a href="#">HOME</a></li>
+                <li><a href="#">CAKES</a></li>
+                <li><a href="#">ORDERIN</a></li>
+                <li><a href="#">LESSONS</a></li>
+            </ul>
+        </section>
+        <section class="middlesection">
+            <div class="divsection">
+                <p class="maintext1">Welcome</p>
+                <p class="maintext2">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Veritatis, suscipit? Rem saepe pariatur perferendis, necessitatibus nisi voluptatibus repellat possimus dolorum. Incidunt eos aut voluptatum minus dicta eius doloribus in assumenda!</p>
+            </div>
+            <img class="imgsection"  src="https://cdn.shopify.com/s/files/1/0002/7803/6521/products/Wedding_FinalWhite_800x.jpg" alt="wedding cake">
+        </section>
+        <section class="bottomsection" >
+            <a class="alink" href="#"><img class="produce"  src="https://t8x8a5p2.stackpathcdn.com/wp-content/uploads/2018/05/Birthday-Cake-Recipe-Image-720x720.jpg" alt="birthday cake"></a>
+            <a class="alink" href="#"><img class="produce" src="https://www.christinebailey.co.uk/wp-content/uploads/2015/01/chocolate-cake-slice-e1448521562644.jpg" alt="cake slice"></a>
+            <a class="alink" href="#"><img class="produce" src="https://www.tasteofhome.com/wp-content/uploads/2019/12/Oreo-Cupcakes-with-Cookies-and-Cream-Frosting_EXPS_FT19_247265_F_1203_1.jpg" alt="cup cake"></a>
+            <a class="alink" href="#"><img class="produce" src="https://www.foodheavenmag.com/wp-content/uploads/2020/03/Richard-Bertinet-Cinnamon-Buns.jpg" alt="cinnamon roll"></a>
+        </section>
+    </main>
+    <footer>
+        <section>
```

Comment:
You probably don't need this `section` - you can put your styles directly on the `footer`